prime/script/check_repo.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# This script checks if current branch is master
+
+if [ master != $1 ];
+then
```

Comment:
I usually prefer the style "if [ master != $1 ] ; then"  (not breaking the line before "then")  and also to wrap the $1 in quotes "$1", just to make sure that the expansion is not treated as code.     Details, but still ... :-)